go.mod

```diff
@@ -1,13 +1,16 @@
 module github.com/smallstep/scep
 
-go 1.16
+go 1.20
 
 require (
 	github.com/go-kit/kit v0.4.0
+	go.mozilla.org/pkcs7 v0.0.0-20210826202110-33d05740a352
+)
+
+require (
 	github.com/go-logfmt/logfmt v0.5.1 // indirect
 	github.com/go-stack/stack v1.6.0 // indirect
-	go.mozilla.org/pkcs7 v0.0.0-20210826202110-33d05740a352
 )
 
 // use github.com/smallstep/pkcs7 fork with patches applied
-replace go.mozilla.org/pkcs7 => github.com/smallstep/pkcs7 v0.0.0-20230302202335-4c094085c948
+replace go.mozilla.org/pkcs7 => github.com/smallstep/pkcs7 v0.0.0-20230615175518-7ce6486b74eb
```

Comment:
Makes sense to me. Probably pointed to the last commit before it was merged into `main` in that repo. Actually in `master` at the time, but I renamed that to `main` this weekend. I did some branch cleanup yesterday.

I've opened https://github.com/smallstep/pkcs7/issues/14 to remind myself we want to rename the module and create a release.